README.rst

```diff
@@ -1,148 +1,186 @@
+Components
+----------
 
-- Fixing context value. NGSI parameter is called value in the latest
-  release of NGSI. [Guillermo Jimenez Prieto]
+Fiware-Cloto
+    Fiware-cloto is part of FIWARE Policy Manager. It provides an API-REST to create rules associated to servers,
+    subscribe servers to Context Broker to get information about resources consumption of that servers and launch actions
+    described in rules when conditions are given.
 
-- Context value parameter is called value in the last specification of
-  NGSI. [Guillermo Jimenez Prieto]
+Fiware-Facts
+    Server to process the incoming facts from the
+    `Orion Context Broker <http://catalogue.fi-ware.org/enablers/publishsubscribe-context-broker-orion-context-broker>`__
+    and publish the result into a RabbitMQ queue to be analysed by Fiware-Cloto. The facts are the result of the server
+    resources consumption.
 
-v1.4.0 (2015-06-29)
--------------------
+For more information, please refer to the `documentation <doc/README.rst>`_.
 
-Fix
-~~~
+Build and Install
+=================
 
-- Checking if window size is instance of integer or long. [Guillermo
-  Jimenez Prieto]
+Requirements
+------------
 
-- Fixing tests about getting windowsizes from cloto component.
-  [Guillermo Jimenez Prieto]
+- Operating systems: CentOS (RedHat) and Ubuntu (Debian), being CentOS 6.3 the
+  reference operating system.
 
-- Fixing Facts component cannot access to the tenant Windowsize without
-  a valid tenant token via API. [Guillermo Jimenez Prieto]
-
-- Adding new variables to the configuration for the new windowsize
-  retriving method. [Guillermo Jimenez Prieto]
-
-- Checking that connection to rabbitMQ is open before close it.
-  [Guillermo Jimenez Prieto]
-
-- Fixing some bugs with the windowsize representation. [Guillermo
-  Jimenez Prieto]
-
-- Checking that there is a connection before close it. [Guillermo
-  Jimenez Prieto]
-
-v1.3.0 (2015-06-01)
--------------------
+To install this module you have to install some components:
 
-New
-~~~
+- Python 2.7
+- Fiware-cloto module (https://github.com/telefonicaid/fiware-cloto)
+- Redis 2.9.1 or above
+- RabbitMQ Server 3.3.0 or above (http://www.rabbitmq.com/download.html)
 
-- Added Windows Size support for tenants. [Guillermo Jimenez Prieto]
+Installation
+------------
+Download the component by executing the following instruction:
 
-Fix
-~~~
+git clone git@github.com:telefonicaid/fiware-facts.git
 
-- Updated Readme File with new installation information. [Guillermo
-  Jimenez Prieto]
+Note: we recommend you to download this component into this location:
+``/opt/policyManager``
 
-v1.2.0 (2015-04-01)
--------------------
 
-New
-~~~
+Configuration file
+------------------
+The configuration used by the fiware-facts component is optionally read from the file
+``conf/fiware-facts.cfg``
 
-- New: dev: preparing release @release. [Guillermo Jimenez Prieto]
+Default values are found in ``facts/config.py``.
```

Comment:
You should explain the main configuration parameters of this file that the user should setup.